<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QCalendarWidget" name="calendarWidget"/>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QWebView" name="webView">
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QWebView" name="webView_2">
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="3" column="0">
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="page">
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="geometry">
               <rect>
                <x>170</x>
                <y>110</y>
                <width>113</width>
                <height>25</height>
               </rect>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="page_3">
             <widget class="QDateTimeEdit" name="dateTimeEdit">
              <property name="geometry">
               <rect>
                <x>170</x>
                <y>150</y>
                <width>194</width>
                <height>25</height>
               </rect>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="page_2">
             <widget class="QCommandLinkButton" name="commandLinkButton">
              <property name="geometry">
               <rect>
                <x>200</x>
                <y>120</y>
                <width>179</width>
                <height>41</height>
               </rect>
              </property>
              <property name="text">
               <string>CommandLinkButton</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>prev</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>next</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_7">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionWeb"/>
   <addaction name="actionPlap"/>
   <addaction name="actionLoop"/>
   <addaction name="actionOngprec"/>
   <addaction name="actionOngsuiv"/>
  </widget>
  <action name="actionWeb">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>activer web</string>
   </property>
   <property name="toolTip">
    <string>obvious</string>
   </property>
  </action>
  <action name="actionPlap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>plap</string>
   </property>
   <property name="toolTip">
    <string>hohoho</string>
   </property>
  </action>
  <action name="actionLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>loop</string>
   </property>
   <property name="toolTip">
    <string>c'est la vie!</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+L</string>
   </property>
  </action>
  <action name="actionOngsuiv">
   <property name="text">
    <string>ongsuiv</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgDown</string>
   </property>
  </action>
  <action name="actionOngprec">
   <property name="text">
    <string>ongprec</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgUp</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
